src/main/webapp/app/overview/postings/answer-post/answer-post.component.ts

```diff
@@ -26,23 +26,23 @@ export class AnswerPostComponent implements OnInit {
     @Input() user: User;
     @Input() isAtLeastTutorInCourse: boolean;
     @Output() interactAnswer = new EventEmitter<AnswerPostAction>();
-    editText?: string;
+    content?: string;
     isLoading = false;
     isEditMode: boolean;
     EditorMode = EditorMode;
     courseId: number;
 
-    // Only allow certain html tags and no attributes
-    allowedHtmlTags: string[] = ['a', 'b', 'strong', 'i', 'em', 'mark', 'small', 'del', 'ins', 'sub', 'sup', 'p'];
-    allowedHtmlAttributes: string[] = ['href'];
+    // Only allow certain html tags and attributes
+    allowedHtmlTags: string[] = ['a', 'b', 'strong', 'i', 'em', 'mark', 'small', 'del', 'ins', 'sub', 'sup', 'p', 'ins', 'blockquote', 'pre', 'code', 'span', 'li', 'ul'];
```

Comment:
`ins` was already in the list. I am not an HTML expert, but the tags look safe to me.
```suggestion
    allowedHtmlTags: string[] = ['a', 'b', 'strong', 'i', 'em', 'mark', 'small', 'del', 'ins', 'sub', 'sup', 'p', 'blockquote', 'pre', 'code', 'span', 'li', 'ul'];
```
We could maybe even allow more tags that are similar, like `ol` for ordered lists.

Reply:
9cc3183